openpype/modules/deadline/plugins/publish/validate_deadline_connection.py

```diff
@@ -1,30 +1,31 @@
 import pyblish.api
 
 from avalon.vendor import requests
-from openpype.plugin import contextplugin_should_run
+from openpype.modules import ModulesManager
 import os
 
 
-class ValidateDeadlineConnection(pyblish.api.ContextPlugin):
+class ValidateDeadlineConnection(pyblish.api.InstancePlugin):
     """Validate Deadline Web Service is running"""
 
     label = "Validate Deadline Web Service"
     order = pyblish.api.ValidatorOrder
     hosts = ["maya", "nuke"]
     families = ["renderlayer"]
 
-    def process(self, context):
-
-        # Workaround bug pyblish-base#250
-        if not contextplugin_should_run(self, context):
-            return
-
-        deadline_url = (
-            context.data["system_settings"]
-            ["modules"]
-            ["deadline"]
-            ["DEADLINE_REST_URL"]
-        )
+    def process(self, instance):
```

Comment:
Could we add deadline collector to collect the deadline urls?

Reply:
done for the default one. What server to use should be resolved before that, possibly in Creators